homeassistant/components/template/strings.json

```diff
@@ -1,4 +1,105 @@
 {
+  "config": {
+    "step": {
+      "sensor": {
+        "data": {
+          "device_class": "Device class",
+          "name": "[%key:common::config_flow::data::name%]",
+          "state_class": "[%key:component::sensor::entity_component::_::state_attributes::state_class::name%]",
+          "state_template": "State template",
+          "unit_of_measurement": "Unit of measurement"
+        },
+        "title": "Template sensor"
+      },
+      "user": {
+        "description": "This helper allow you to create helper entities that define their state using a template.",
+        "menu_options": {
+          "sensor": "Template a sensor"
+        },
+        "title": "Template helper"
+      }
+    }
+  },
+  "options": {
+    "step": {
+      "sensor": {
+        "data": {
+          "device_class": "Device class",
```

Comment:
Why why aren't state class and unit part of the data here?

Reply:
Fixed!